### PR TITLE
[TACHYON-1323] Changed code in Javadoc in different modules

### PR DESCRIPTION
**Pull request by HorizonNet**
https://tachyon.atlassian.net/browse/TACHYON-1323

### CODE REVIEW

File: fuse/src/main/java/tachyon/fuse/OpenFileEntry.java [hunk 1]
```diff
@@ -42,10 +42,9 @@ public OpenFileEntry(FileInStream in, FileOutStream out) {
   }
 
   /**
-   * Gets the opened input stream for this open file entry.
+   * Gets the opened input stream for this open file entry.  The value returned can be {@code null}
```
Comment (aaudiber) [inline fuse/src/main/java/tachyon/fuse/OpenFileEntry.java]: remove extra space after period, here and below

File: integration-tests/src/test/java/tachyon/client/RemoteBlockInStreamIntegrationTest.java [hunk 2]
```diff
@@ -543,7 +543,7 @@ public void seekAroundLocalBlock() throws IOException, TachyonException {
   }
 
   /**
-   * Test remote read stream lock in <code>RemoteBlockInStream</code>.
+   * Test remote read stream lock in {@code RemoteBlockInStream}.
```
Comment (aaudiber) [inline integration-tests/src/test/java/tachyon/client/RemoteBlockInStreamIntegrationTest.java]: Test -> Tests

`@code` -> `@link`

**Comment by aaudiber** [comment]
Thanks for the doc improvements! I've left some comments

**Comment by HorizonNet** [comment]
@aaudiber Thanks for the comments. I updated the PR accordingly.

**Comment by aaudiber** [comment]
LGTM

**Pull request merged**